src/molecules/Select/Input.test.tsx

```diff
@@ -28,9 +30,8 @@ describe('Select <Input />', () => {
 
   it('it has the correct style properties when isOpen is false', () => {
     let wrapper = setup(false);
-    let input = wrapper;
```

Comment:
just small cleanup to make consistent naming with other tests. Also sounds like wrapper is the preferred nomenclature.